test/mpi/maint/dtp-test-config.txt

```diff
@@ -13,45 +13,40 @@
 # NOTE: the second (ssv args), fourth (timeLimit) and seventh (maxtestsize) fields are optional.
 
 attr/fkeyvaltype::1::1:1024:
-coll/bcast::1 512 262144:timeLimit=420:4:16:1024
-coll/bcast_comm_world_only::1 512 262144:timeLimit=360:10:16:1024
+coll/bcast::1 512::4:16:1024
+coll/bcast::262144:timeLimit=420 mem=1.2:4:16:
+coll/bcast_comm_world_only::1 512::10:16:1024
+coll/bcast_comm_world_only::262144:timeLimit=360 mem=1.9:10:16:
```

Comment:
I thought we discussed modifying the comment header in this file to describe the `mem=` format?

Reply:
I remember I did. But apparently I was somehow distracted and the commit (amend) seems to be lost. Anyway, added a new commit for that.